pkg/kubelet/kubelet_getters.go

```diff
@@ -142,7 +142,13 @@ func (kl *Kubelet) getPodContainerDir(podUID types.UID, ctrName string) string {
 // GetPods returns all pods bound to the kubelet and their spec, and the mirror
 // pods.
 func (kl *Kubelet) GetPods() []*v1.Pod {
-	return kl.podManager.GetPods()
+	pods := kl.podManager.GetPods()
+	for _, p := range pods {
```

Comment:
nit: can you add a comment explaining that this is required to get the status of static pods when there is no apiserver?